WPMedia/fusion-news-theme-blocks
# Pen 1459 video promo live

## Description
- update the live customfields from checkbox to text input
- if input box is empty, live label should not be shown

## Jira Ticket
- [PEN-1459](https://arcpublishing.atlassian.net/browse/PEN-1459)

## Acceptance Criteria
On the Video Promo Block, remove the existing Live boolean Custom Field, and instead implement a string (text) custom field called Alert badge 

If a value is entered in this field, that exact text should appear above the video (aligned left), using the label specs in the Zeplin designs (using the Danger red color, which is also used for the Alert Bar) 

Note – the text should render exactly as it is input into the Custom Field – don’t automatically capitalize it or anything like that

If no value is entered in this field, then the Alert badge should not appear at all

## Test Steps
1. on the theme repo, update .env file:
- [ ] link your local engine-theme-sdk, e.g. ENGINE_SDK_REPO={PATH/TO/LOCAL/engine-theme-sdk}
- [ ] add videoOrg=prod
- [ ] add videoOrg=corecomponents

2. under block repo
create a new page and add video promo block

3. test the `Live` customfield, if no value is entered, the label should not be shown

4. entered a sample value to the `Live` field, the text should match on the label without any capitalization etc
<img width="1007" alt="Screen Shot 2020-11-06 at 11 00 21 AM" src="https://user-images.githubusercontent.com/4411687/98420273-aea2db80-203b-11eb-89f0-25ac74ae11a6.png">


## Effect Of Changes
### Before
- `Live` customfield is a checkbox:
<img width="455" alt="Screen Shot 2020-11-06 at 2 29 02 PM" src="https://user-images.githubusercontent.com/4411687/98420669-8667ac80-203c-11eb-9985-fb79e74a5438.png">


### After
- `Live` customfield updated to text input box
<img width="662" alt="Screen Shot 2020-11-06 at 11 00 31 AM" src="https://user-images.githubusercontent.com/4411687/98420785-c9c21b00-203c-11eb-9545-e883ff64cdf6.png">


## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
